homeassistant/components/camera/media_source.py

```diff
@@ -46,15 +46,17 @@ async def async_resolve_media(self, item: MediaSourceItem) -> PlayMedia:
                 f"/api/camera_proxy_stream/{camera.entity_id}", camera.content_type
             )
 
-        if stream_type != StreamType.HLS:
-            raise Unresolvable("Camera does not support MJPEG or HLS streaming.")
-
         if "stream" not in self.hass.config.components:
             raise Unresolvable("Stream integration not loaded")
 
         try:
             url = await _async_stream_endpoint_url(self.hass, camera, HLS_PROVIDER)
```

Comment:
I'm on my phone, what does the error handling do here?

Reply:
I suppose it was meant to return a meaningful message to end-user (which this PR keeps).